docker/py/Dockerfile

```
@@ -29,6 +29,9 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     python-dev \
     unzip \
     vim && \
+    wget https://github.com/cdr/code-server/releases/download/v3.6.2/code-server_3.6.2_amd64.deb && \
+    dpkg -i ./code-server*.deb && \
+    rm code-server_3.6.2_amd64.deb && \
```

```suggestion
    rm code-server*.deb && \
```

I think you should keep the `*` in the rm command as well as the `dpkg` command. Or use the specific version in both. Did you add the `*` in case the version changes in the future?

---

The specific version is still hardcoded in the `wget` call btw.

---

good point - I actually meant to make the version a top level argument so it can be changed easily everywhere. Would that make sense you think?

---

Yeah I think making the image a top level argument is best.

---

This is not relevant anymore

---

right I'll add the argument to the template Dockerfile where it can be changed. I did make it a variable in the script.